webapp/components/product-info.js

```diff
@@ -38,7 +38,7 @@ const versionPatterns = Object.freeze({
 });
 
 // The set of all browsers known to the wpt.fyi UI.
-const AllBrowserNames = Object.freeze(['chrome', 'deno', 'edge', 'firefox', 'safari', 'servo', 'webkitgtk']);
+const AllBrowserNames = Object.freeze(['chrome', 'edge', 'firefox', 'safari', 'servo', 'webkitgtk', 'deno']);
```

Comment:
Rather than making this non-alphabetical, can the tests be fixed?

Reply:
yup commented above

Reply:
Oops :)

Reply:
So just to be explicit, I'm waiting for another push before reviewing.

Reply:
Pushed!